src/components/letter/LetterDetailBottomSheet.tsx

```
@@ -1,112 +1,75 @@
 
   return (
-    <div className="fixed inset-0 z-50 flex justify-center">
-      <div className="relative w-full max-w-[440px] min-h-screen">
-        <button
-          onClick={onClose}
-          className="absolute inset-0 bg-black/40"
-        />
-
-        <div className="absolute bottom-0 w-full bg-white rounded-t-2xl py-[62px] flex flex-col gap-[40px]">
+    <>
+      <BottomSheet open={open} onClose={onClose} className="px-[20px]" contentClassName="gap-[12px]">
```

ESLint

[error] Replace `·open={open}·onClose={onClose}·className="px-[20px]"·contentClassName="gap-[12px]"` with `⏎········open={open}⏎········onClose={onClose}⏎········className="px-[20px]"⏎········contentClassName="gap-[12px]"⏎······`

(prettier/prettier)